### PR TITLE
structured out folders for UI components

### PR DESCRIPTION
**Pull request by milenapribic**
Take 2

### CODE REVIEW

File: client/components/ScheduleList/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Identity from '../index';
```
Comment (chrisdhanaraj) [inline client/components/ScheduleList/stories/index.js]: should be ScheduleList

**Comment by chrisdhanaraj** [comment]
👍 but swap to stateless functional components and fix that issue with the import on the ScheduleList

**Comment by milenapribic** [comment]
just fixed issue for ScheduleList and rewrote the index files

**Comment by chrisdhanaraj** [comment]
Almmoost, you don't need the { Component } part of the import anymore

**Comment by milenapribic** [comment]
THANKS! Removed it meow

**Comment by chrisdhanaraj** [comment]
👍 merging it

**Pull request merged**